.idea/workspace.xml

```diff
@@ -1,7 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ovaj fajl ti ne treba u repositoriju. To je file vezan samo za tvoj IntellJ i ništa neće  koristiti korisnicima tvog repositorija.

Reply:
ok, njega uklonim